components/reddit/reddit.app.js

```diff
@@ -246,9 +249,9 @@ module.exports = {
             title,
             name,
           });
-        });
+        }); 
```

Comment:
Same as before regarding the trailing whitespaces

Reply:
Ok this is fixed.